_posts/2021-04-25-ocp-java11.md

```diff
@@ -0,0 +1,225 @@
+hit the nail there. Sometimes you might be just randomly right, and I avoided false positives. Of course, this takes time then.
+
+Overall the mock exams do quite well simulate the real exam in their content and extend. You'll get a set of ~50 questions distributed over all topics,
+just like in the exam. I did not enforce a time box of 90 mins like in the exam. Estimated each mock exam including reviews took 2 - 2,5h.
+
+# Exam Check-in
+
+Finally the exam day came. It was time to check in. You'll receive a week before a mail from Pearson VUE asking you to [check the system requirements
+and exam policies](https://home.pearsonvue.com/oracle/onvue). For the system test you'll need to download the testing software, which requires that you don't have any other program
+running (except for Finder on macOS). Make sure to read [Tips on taking an OnVUE exam](https://home.pearsonvue.com/onvue-tips). Honestly, I did not 
+do this and it would have avoided some stress. Let me tell what went wrong for me.
+
+I started the login procedure as recommended 30 minutes before planned start. You'll need to start the OnVUE software and you'll get a code from the login link 
+in the email you received. Before I mostly cleaned my desk except for the external monitor I wanted to work with my Notebook. You need to set the camera
+you are using at first. For me, I had the notebook closed and chose the external webcam. The software showed me then the camera stream, so it looked
+fine so far.
+
+The next step involved using the smartphone for the check-in procedure. You have 2 options to continue: Either get an SMS with a link, or enter the link yourself.
+I chose the SMS, but unfortunately I did not receive the message. The OnVUE software was waiting until you continue on the smartphone and I was blocked.
+I needed to restart and the software and then entered the link manually. Here a QR code would have helped a bit.
+
+I opened a link and it got some information. One of them was that no external monitors are allowed. I wanted to avoid problems and unplugged mine, switching
+to the notebook's display and camera. If I would have read the tips carefully then there you read that an external monitor is actually allowed
+when the notebook is closed. I was just confused by the message on the web page. Then the check-in software requested to make a photo of yourself, then front and back of your identity document and photos of your room in front of desk, 
+back, left and right.
+
+Finally I was done there and got into the waiting queue. There was a counter how many people are checked in before you. I got about position 25, and it
+took between 5 and 10 minutes until I was in front.
+
+Now a proctorer opened a chat window and wrote that he could not see my video stream. This will be surely because I unplugged the external monitor and
```

Comment:
I would use proctor instead of proctorer (I can't see if this word exists).

Reply:
oh right ;)